r_py_cheatsheet.md

```diff
@@ -405,17 +405,21 @@ Working example (asap):
 
 ### R
 
-Purpose:
-Official documentation:
-Good help, tutorials:
+Purpose: spread() takes two columns (key & value), and spreads into multiple columns: it makes “long” data wider.
 
-1. ...
-1. ...
-1. ...
+Official documentation: https://www.rdocumentation.org/packages/tidyr/versions/0.8.3/topics/spread
+Good help, tutorials:
+1. https://r4ds.had.co.nz/tidy-data.html#spreading
 
 Working example (asap):
 
 ```R
+
+library(tidyverse)
+#print(table2)
```

Comment:
What is the table2 object?
Please change the code so that it is using a real table with some contents. 
You can either make one up, or use example data from an existing dataset. 
Thank you :-)

Reply:
The dataset shows the values of four variables country, year, population, and cases which comes from tidyverse library. When you paste this code in your r studio you should get the following output:

<img width="327" alt="Bildschirmfoto 2019-05-22 um 10 48 08" src="https://user-images.githubusercontent.com/49499092/58160612-1b2aee80-7c7f-11e9-9a17-84fccec21241.png">

Reference: https://r4ds.had.co.nz/tidy-data.html#prerequisites-6

Reply:
Very good! I had no idea.
Thank you.